docs/analysing_tardis_outputs/visualization/how_to_liv_plot.ipynb

```diff
@@ -17,9 +17,22 @@
   },
```

Comment:
<p>Why have you committed this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/3018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>